src/minuteman_metrics.erl

```diff
@@ -377,8 +375,8 @@ process_p99s_4_test_() ->
 process_dst_ip_map_test_() ->
     DAddr = {54, 192, 147, 29},
     IP = 16#36c0931d,
-    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, IP, 8080, 167792566, 8081, 59},
-    Conn2 = {ip_vs_conn, tcp, established, 167792567, 47808, IP, 8080, 167792566, 8081, 59},
+    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, 167792566, 8080, IP, 8081, 59},
```

Comment:
yeah, but records are just syntactic sugar on top of tuples. Ultimately, the two representations are the same. Interesting would be to see if we miss something in one representation as compare to other. Do you think that could be the case?